pages/[lang]/index.tsx

```diff
@@ -108,7 +108,7 @@ export default function HomePage() {
 
               <div className='mt-12 lg:flex lg:flex-row-reverse'>
                 <section className='mt-10 lg:mt-0 lg:flex-1'>
-                  <Calendar size={2} className='float-left' />
+                  <Calendar size={2} className='content-center' />
```

Comment:
In this way, all the content inside component will be centred, but we only want the content to be centred, when there are no meetings present.